compose.yml

```diff
@@ -11,12 +11,14 @@ services:
       - mosquitto
     network_mode: "host"
     volumes:
-      - ./eos_artifacts:/usr/src/app/eos_artifacts
+      - /media/card/eos_artifacts:/usr/src/app/eos_artifacts
       - .:/usr/src/app:ro
       - /run/udev:/run/udev:ro
+    restart: always
```

Comment:
This looks like it checks out but per [this article](https://jason19970210.medium.com/a-step-by-step-how-to-start-docker-compose-container-at-boot-effe9964e57f), but could you verify this behaves expectedly on the bone?

Reply:
Already done